src/cpp/security/accesscontrol/Permissions.cpp

```diff
@@ -766,6 +828,13 @@ PermissionsHandle* Permissions::validate_local_permissions(
         return nullptr;
     }
 
+    bool transmit_legacy_algorithms = false;
```

Comment:
@MiguelCompany Should we change the default value in the backports ? Or we can leave it as it is